### DISCUSSION TITLE
About the difference between my code and the example

### DISCUSSION THREAD

**Discussion by GoldenGrailJJ** [THUDM/CogDL#389] [General]
![image](https://user-images.githubusercontent.com/33853765/188374856-94084349-da48-4b4f-9b6f-6a6784860978.png)

![image](https://user-images.githubusercontent.com/33853765/188374909-47c03d8c-8bbd-4ea9-9c34-d7682d4f250b.png)

Why the result of the first picture is different with the result in the second picture?

**Comment by QingFei1** [comment]
Sorry, the result of the example is wrong.We will update the document later
